.travis.yml

```diff
@@ -57,8 +59,27 @@ matrix:
     - compiler: ghc-head
     - compiler: ghc-8.8.1
 before_install:
+  - |
+    if [ "$TRAVIS_OS_NAME" = "linux" ]; then
+      if $GHCJS ; then sudo add-apt-repository -y ppa:hvr/ghcjs ; fi;
+      if $GHCJS ; then curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add - ; fi;
+      if $GHCJS ; then sudo apt-add-repository 'https://deb.nodesource.com/node_8.x xenial main' ; fi;
+      sudo apt-get update;
+      sudo apt-get install $CC freeglut3-dev;
+      if $GHCJS ; then sudo apt-get install -y nodejs cabal-install-3.0 ; fi;
+    fi
   - HC=$(echo "/opt/$CC/bin/ghc" | sed 's/-/\//')
   - WITHCOMPILER="-w $HC"
+  - |
+    if echo $CC | grep -q ghcjs; then
+        GHCJS=true
+        HC=${HC}js
+        WITHCOMPILER="--ghcjs ${WITHCOMPILER}js"
+    else
+        GHCJS=false;
+    fi
+  - if $GHCJS ; then sudo apt-get install -y ghc-8.4.4 ; fi
```

Comment:
Should we be hardcoding `ghc-8.4.4` like this? If we ever support multiple GHCJS versions, it seems like we would need to use something like `ghc-${GHCJSNUMVER}`, right?

Reply:
It's an issue to solve when there's some other GHCJS than 8.4 in hvr's (or actually any) apt repository.